lib/std/math.zig

```diff
@@ -270,6 +270,41 @@ pub const sinh = @import("math/sinh.zig").sinh;
 pub const cosh = @import("math/cosh.zig").cosh;
 pub const tanh = @import("math/tanh.zig").tanh;
 
+/// Sine trigonometric function on a floating point number.
+/// Uses a dedicated hardware instruction when available.
+/// This is an alias to the builtin @sin
```

Comment:
This is technically not true, it's not an alias. I would reword this to "This is the same as calling..."